packages/core/cli.ts

```diff
@@ -191,13 +191,14 @@ yargs(process.argv.slice(2))
 					group: "Fuzzer:",
 					default: 5000,
 				})
-				.array("disable_bug_detectors")
-				.option("disable_bug_detectors", {
+				.array("disableBugDetectors")
```

Comment:
Because of how we load the Jest config---the names in our `Options` _are_ our Jest config keys. Either we want camelCased key names that we use everywhere in our code (e.g. `options.disableBugDetectors`), or we want them snake cased. Conversely, since all our `Option` keys are camelCased, we already have options that are inconsistent between the CLI and Jest mode (e.g. `custom_hooks` on CL vs. `customHooks` in Jest mode). I noticed this problem recently, and have added a ticket for that already :smiley:

Reply:
Then I misunderstood the ticket. IMHO it's ok to use different names for parameters on the CLI and in JavaScript config files, as different conventions are at play here. I would tend to stick with snake case on the CLI, to mimic libFuzzer parameters, and camel case in JavaScript files.